core/internal/cltest/cltest.go

```diff
@@ -1687,3 +1687,11 @@ func MustGetStateForKey(t testing.TB, kst keystore.Eth, key ethkey.KeyV2) ethkey
 	require.NoError(t, err)
 	return states[0]
 }
+
+func NewBulletproofTxManagerORM(t *testing.T, db *sqlx.DB) bulletprooftxmanager.ORM {
+	return bulletprooftxmanager.NewORM(db, logger.TestLogger(t))
+}
+
+func MustExec(t *testing.T, db *sqlx.DB, stmt string, args ...interface{}) {
```

Comment:
Could we move this to `core/internal/testutils/pgtest` so that we have less reliance on `cltest`?